common/test/acceptance/pages/lms/discussion.py

```diff
@@ -733,3 +733,11 @@ def get_new_post_preview_text(self):
         """
         self.wait_for_element_visibility(".wmd-preview > div", "WMD preview pane has contents", timeout=10)
         return self.q(css=".wmd-preview").text[0]
+
+    def get_preview_value(self):
```

Comment:
What if we refactor `get_new_post_preview_value` in a way that it can be used for intended purpose here?